plugins/ssm-secret.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ssm-secret
+spec:
+  version: v1.2.1
+  homepage: https://github.com/pr8kerl/kubectl-ssm-secret
+  caveats: |
+    Read the documentation at https://github.com/pr8kerl/kubectl-ssm-secret.
+    You can call this plugin with "kubectl ssm-secret".
+  shortDescription: import/export kubernetes secrets from/to AWS SSM parameter store paths.
+  description: |
+    This plugin can import (create/update) a kubernetes secret with key/values pairs stored in AWS SSM parameter store.
+    It can also export kubernetes secret data to an AWS SSM parameter store path.
+  platforms:
+  - uri: https://github.com/pr8kerl/kubectl-ssm-secret/releases/download/v1.2.1/kubectl-ssm-secret_1.2.1_darwin_x86_64.tar.gz
+    sha256: 1e76ef636ae1bcd6c854059d37956b65302c4cc5349a9a59979eb25007279bf8
+    bin: kubectl-ssm_secret
+    files:
+    - from: "*"
+      to: "."
```

Comment:
this value is actually the default.
you can drop `files:` section altogether if this is what you do.

Reply:
Thanks. I was using an older version of krew. Now removed and using latest krew - seems good.